src/store/useAppStore.ts

```
@@ -665,6 +696,8 @@ export const useAppStore = create<AppState & AppActions>()(
       releases: [],
       releaseSubscriptions: new Set<number>(),
       readReleases: new Set<number>(),
+      readForks: new Set<number>(),
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)